cnd/src/db/schema.rs

```diff
@@ -1,106 +1,179 @@
+        bitcoin_network -> Text,
+        ethereum_chain_id -> BigInt,
+        bitcoin_amount -> Text,
+        erc20_amount -> Text,
+        erc20_token_contract -> Text,
+        hash_function -> Text,
+        bitcoin_refund_identity -> Text,
+        ethereum_redeem_identity -> Text,
+        bitcoin_expiry -> BigInt,
+        ethereum_expiry -> BigInt,
+        secret_hash -> Text,
+    }
 }
 
 table! {
-   rfc003_ethereum_bitcoin_erc20_bitcoin_request_messages {
-       id -> Integer,
-       swap_id -> Text,
-       ethereum_chain_id -> BigInt,
-       bitcoin_network -> Text,
-       erc20_amount -> Text,
-       erc20_token_contract -> Text,
-       bitcoin_amount -> Text,
-       hash_function -> Text,
-       ethereum_refund_identity -> Text,
-       bitcoin_redeem_identity -> Text,
-       ethereum_expiry -> BigInt,
-       bitcoin_expiry -> BigInt,
-       secret_hash -> Text,
-   }
+    rfc003_ethereum_bitcoin_erc20_bitcoin_request_messages {
+        id -> Integer,
+        swap_id -> Text,
+        ethereum_chain_id -> BigInt,
+        bitcoin_network -> Text,
+        erc20_amount -> Text,
+        erc20_token_contract -> Text,
+        bitcoin_amount -> Text,
+        hash_function -> Text,
+        ethereum_refund_identity -> Text,
+        bitcoin_redeem_identity -> Text,
+        ethereum_expiry -> BigInt,
+        bitcoin_expiry -> BigInt,
+        secret_hash -> Text,
+    }
 }
 
 table! {
-   rfc003_ethereum_bitcoin_accept_messages {
-       id -> Integer,
-       swap_id -> Text,
-       ethereum_redeem_identity -> Text,
-       bitcoin_refund_identity -> Text,
-       at -> Timestamp,
-   }
+    rfc003_ethereum_bitcoin_accept_messages {
+        id -> Integer,
+        swap_id -> Text,
+        ethereum_redeem_identity -> Text,
+        bitcoin_refund_identity -> Text,
+        at -> Timestamp,
+    }
 }
 
 table! {
-   rfc003_bitcoin_ethereum_accept_messages {
-       id -> Integer,
-       swap_id -> Text,
-       bitcoin_redeem_identity -> Text,
-       ethereum_refund_identity -> Text,
-       at -> Timestamp,
-   }
+    rfc003_bitcoin_ethereum_accept_messages {
+        id -> Integer,
+        swap_id -> Text,
+        bitcoin_redeem_identity -> Text,
+        ethereum_refund_identity -> Text,
+        at -> Timestamp,
+    }
 }
 
 table! {
-   rfc003_decline_messages {
-       id -> Integer,
-       swap_id -> Text,
-       reason -> Nullable<Text>,
-   }
+    rfc003_decline_messages {
+        id -> Integer,
+        swap_id -> Text,
+        reason -> Nullable<Text>,
+    }
 }
 
 table! {
-   rfc003_swaps {
-       id -> Integer,
-       swap_id -> Text,
-       role -> Text,
-       counterparty -> Text,
-   }
+    rfc003_swaps {
+        id -> Integer,
+        swap_id -> Text,
+        role -> Text,
+        counterparty -> Text,
+    }
+}
+
+// The new split protocol tables i.e., Han, HErc20, HALight.
+//
+
+// This is used by Alice to save dial info for created swaps.
+table! {
+    dial_infos {
```

Comment:
Not sure if we had this before like that but we could name this one swaps and include the role as-well. Then the individual tables would only contain ledger specific stuff.

Reply:
Nice, done.